groups/models.py

```diff
@@ -161,6 +167,9 @@ def delete_state(self):
     def is_deleted(self):
         return self.state == self.STATE_DELETED
 
+    def __str__(self):
+        return 'Comment by {}'.format(self.user.get_full_name())
```

Comment:
To be clear: the reason why it's not a good plan to use related model information is that one can inadvertently do another query when showing the object. In a list this can quickly get unwieldy.